src/LaunchDarkly.Client/Feature.cs

```diff
@@ -119,43 +122,52 @@ public bool Matches(User user)
 
         private Object GetUserValue(User user)
         {
-            switch (Attribute)
+            if (!string.IsNullOrEmpty(Attribute))
```

Comment:
I think it would be clearer  to handle the null case and close the conditional block with a return rather than wrapping all the logic in a conditional.

Plus, `if x` is almost always clearer than `if !x`